pkg/resources/sop_url.go

```diff
@@ -2,43 +2,43 @@ package resources
-	SopUrlRHMICloudResourceOperatorMetricsServiceEndpointDown = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/RHMICloudResourceOperatorMetricsServiceEndpointDown.asciidoc"
-	SopUrlRHMICloudResourceOperatorVPCActionFailed            = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/RHMICloudResourceOperatorVPCActionFailed.asciidoc"
-	SopUrlRHMIThreeScaleApicastProductionServiceEndpointDown  = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/RHMIThreeScaleApicastProductionServiceEndpointDown.asciidoc"
-	SopUrlRHMIThreeScaleApicastStagingServiceEndpointDown     = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/RHMIThreeScaleApicastStagingServiceEndpointDown.asciidoc"
-	SopUrlRHMIThreeScaleBackendListenerServiceEndpointDown    = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/RHMIThreeScaleBackendListenerServiceEndpointDown.asciidoc"
-	SopUrlRHMIThreeScaleZyncServiceEndpointDown               = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/RHMIThreeScaleZyncServiceEndpointDown.asciidoc"
-	SopUrlRHMIThreeScaleZyncDatabaseServiceEndpointDown       = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/RHMIThreeScaleZyncDatabaseServiceEndpointDown.asciidoc"
-	SopUrlThreeScaleBackendWorkerPod                          = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/ThreeScaleBackendWorkerPod.asciidoc"
-	SopUrlThreeScaleAdminUIBBT                                = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/ThreeScaleAdminUIBBT.asciidoc"
-	SopUrlThreeScaleDeveloperUIBBT                            = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/ThreeScaleDeveloperUIBBT.asciidoc"
-	SopUrlThreeScaleSystemAdminUIBBT                          = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/ThreeScaleSystemAdminUIBBT.asciidoc"
-	SopUrlPodDistributionIncorrect                            = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/multi-az/pod_distribution.md"
-	SopUrlSloRhssoAvailabilityAlert                           = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/SloRhssoAvailabilityAlert.asciidoc"
-	SopUrlSloUserSsoAvailabilityAlert                         = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/SloUserSsoAvailabilityAlert.asciidoc"
-	SopUrlTestFireAlerts                                      = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/2.x/cssre_info/info_test_fire_alerts.md#resolve-test-alerts"
+	sopUrlRhoamBase                                            = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/"
```

Comment:
seems inserted before "=", and as result - many identical lines shown as different. Maybe it's possible to remove spaces.

Reply:
GoLand is automatically aligning all `=` signs of bulk variable declarations. In this case, the longest variable name increased by 1 character which made all other variables indented by one extra space.